nix/default.nix

```diff
@@ -63,10 +64,12 @@ let
         shellHook =
           let
             nodeModules = mkNodeModules { };
+            ogmiosFixtures = buildOgmiosFixtures { };
           in
           ''
             export NODE_PATH="${nodeModules}/lib/node_modules"
             export PATH="${nodeModules}/bin:$PATH"
+            export OGMIOS_FIXTURES="${ogmiosFixtures}"
```

Comment:
Please remove this and add it to CTL's own `devShell`. Currently, _all_ library users will need to build the Ogmios fixtures even though they probably have no need for it (and even if they do, you're making it available with `buildOgmiosFixtures` below)

You can override the `shellHook` of our `devShell` to do this (the `shellHook` argument won't work since we need the `purescriptProject` to get the `buildOgmiosFixtures`), i.e.

```
          devShell = project.devShell.overrideAttrs (oas: {
            shellHook = oas.shellHook +
              ''
                export OGMIOS_FIXTURES="${buildOgmiosFixtures {}}"
              '';
          });
```

(The `devShell` is currently on line 433 of the flake)

Alternately, you could move the definition of `buildOgmiosFixtures` to outside this module, as it's not a project-wide tool that library users will likely find useful and doesn't depend on anything else in this module (except `projectName`, but that's not truly necessary)